.github/CONTRIBUTING.md

```diff
@@ -371,13 +371,45 @@ This does not go for bug reports however, as we want to unearth and solve as man
 
 ### Commit
 
-* Focus on one thing per commit 
-* Keep changes as small as possible 
-* Only commit related functinoality for each commit 
+You may have encountered statements like
+
+> Commit early, Commit often
+
+and
+
+> Commit Often, Perfect Later, Publish Once
+>
+> _[Git Best Practices](https://sethrobertson.github.io/GitBestPractices/)_
+
+#### The good commit follows Git best practices
+
+You are encouraged to follow commonly adopted Git best practices, see for example [Distributed Git - Contributing to a Project](https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project). These include, among others:
+
+* Try to make each commit a logically separate changeset
+  + Keep changes in each commit to a minimum
+  + Focus on _one thing_ per commit
+  + Only commit related functionality for each commit
+  + If you have made multiple non-related changes to a file and you want them in separate commits, you can stage your changes using `git add --patch`
 * Commit often
-* Strive for each commit to compile/build and pass tests 
-* Use [gitmoji](https://gitmoji.dev) when possible - point to VSCode extension and CLI tool
-* sidenote: when merging the CI pipeline combines each commit to a merge commit containing all commit messages
+  + Multiple smaller commits are better than one large commit
+* Strive for each commit to compile/build and pass tests
+* Regularly push your commits to remote
+  + Early peer reviews are easier
+  + In case something happens to your machine it reduces the amount of work lost
+  + Makes it easier to hand over the issue to other developers if needed
+
+#### The good commit has good commit messages
+
+* Use imperative mood
+* Describe what was done and why, but not how
+* Limit the subject line to 50 characters. If you cannot fit your message into 50 characters:
```

Comment:
I think this list could also use the same formatting makeover, as mentioned in the above comment